promgen/templatetags/promgen.py

```diff
@@ -36,6 +36,7 @@ def rule_dict(rule):
         "for": rule.duration,
         "labels": rule.labels,
         "annotations": rule.annotations,
+        "description": rule.description,
```

Comment:
This may be the wrong place for this, since this is trying to render in the Prometheus format. You should be able to edit the templates where it shows, and specifically show the description as something Promgen specific.

Reply:
It should probably go into the template around here

https://github.com/line/promgen/blob/d028f71346d6664a1cbc75b41db4caaf247ba56c/promgen/templates/promgen/rule_detail.html#L28-L37

Reply:
Fixed in a34503a.